### PR TITLE
C 0.99.14 release prep

### PR DESCRIPTION
**Pull request by benjeffery**

### CODE REVIEW

**Comment by jeromekelleher** [comment]
I'm not sure we're ready for a Python beta here @benjeffery - we've just documented the find_ibd method, but the semantics are actively being updated at the moment and they may well change. Also, the 64 bit changes are very fresh, I'd rather let it sit for a few weeks so that devs have a chance to see if anything peculiar happens?

We could tag a C release though, and see how the changes go down for users like SLiM?

**Comment by benjeffery** [comment]
Sounds good - happy to hold back on the python as the main motivation is to get the C release done.

**Comment by benjeffery** [comment]
Updated to be a C release

File: c/CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ----------------------
-[0.99.14] - 2021-0X-XX
+[0.99.14] - 2021-09-03
 ----------------------
```
Comment (jeromekelleher) [inline c/CHANGELOG.rst:4]: We've got a bunch of breaking changes here, don't we?

**Review by jeromekelleher — COMMENTED** [review]
Looks good, but we need to get the details into the changelog. 

We should also take a look at the 0.99.14 milestone: https://github.com/tskit-dev/tskit/milestone/25

**Comment by benjeffery** [comment]
@mergify rebase

File: c/CHANGELOG.rst (hunk 1 above)
Comment (benjeffery) [reply c/CHANGELOG.rst:4]: Yep, just rebased to pull that in.

**Review by jeromekelleher — APPROVED** [review]

**Pull request merged**